mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -41,6 +41,9 @@ def cross_entropy(pred,
         reduction='none',
         ignore_index=ignore_index)
 
+    if reduction == 'mean' and avg_factor is None:
```

Comment:
Add comments to explain the reason. It is not straightforward to understand that it aims to calculate the real mean value of the loss.

Reply:
Post-related issues.